src/config.ts

```diff
@@ -12,11 +12,17 @@ export interface VueGtmUseOptions {
     gtm_cookies_win: string;
   };
   /**
-   * Script can be set to `defer` to increase page-load-time at the cost of less accurate results (in case visitor leaves before script is loaded, which is unlikely but possible)
+   * Script can be set to `defer` to decrease page-load-time at the cost of less accurate results (in case visitor leaves before script is loaded, which is unlikely but possible). Defaults to false, so the script is loaded `async` by default
    *
    * @default false
    */
   defer?: boolean;
+  /**
+   * Will add `async` and `defer` to the script tag to not block requests for old browsers that do not suppert `async`
```

Comment:
I think you meant `support` :slightly_smiling_face:

Reply:
true :)